src/Knp/Rad/ResourceResolver/ParameterCaster/ServiceCaster.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Knp\Rad\ResourceResolver\ParameterCaster;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class ServiceCaster
+{
+    private $container;
+
+    public function __construct(ContainerInterface $container)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * @return bool
+     */
+    public function supports($string)
+    {
+        return 1 === preg_match('/^@\w+/', $string);
```

Comment:
Didn't works with `@security.context` for exemple. Maybe better to use `^@[\w.-]+`